node_modules/gh-orgunit/tests/test-export.js

```diff
@@ -0,0 +1,260 @@
+                    OrgUnitTestUtil.assertExportOrgUnit(global.tests.admins.cam2013.client, part.id, null, function(data) {
+                        return callback();
+                    });
+                });
+            });
+        });
+    });
+
+    /**
+     * Test that verifies validation when exporting an organisational unit
+     */
+    it('verify validation when exporting an organisational unit', function(callback) {
+        OrgUnitTestUtil.createSimpleTree(global.tests.admins.cam2013.client, function(course, subject, part, module, series) {
+
+            // Invalid organisational unit id
+            OrgUnitTestUtil.assertExportOrgUnitFails(global.tests.admins.cam2013.client, 'not an id', 'json', 400, function() {
+
+                // Unknown organisational unit id
+                OrgUnitTestUtil.assertExportOrgUnitFails(global.tests.admins.cam2013.client, -1, 'json', 404, function() {
+                    OrgUnitTestUtil.assertExportOrgUnitFails(global.tests.admins.cam2013.client, 234234234234, 'json', 404, function() {
+
+                        // Invalid format
+                        OrgUnitTestUtil.assertExportOrgUnitFails(global.tests.admins.cam2013.client, part.id, 'invalid format', 400, function() {
+                            return callback();
+                        });
+                    });
+                });
+            });
+        });
+    });
+
+    /**
+     * Test that verifies authorization when exporting an organisational unit
+     */
+    it('verify authorization when exporting an organisational unit', function(callback) {
+        OrgUnitTestUtil.createSimpleTree(global.tests.admins.cam2013.client, function(course, subject, part, module, series) {
+
+            // Anonymous users cannot export an organisational unit
+            TestsUtil.getAnonymousAppUserClient(global.tests.apps.cam2013, function(anonymousClient) {
+                OrgUnitTestUtil.assertExportOrgUnitFails(anonymousClient, part.id, null, 401, function() {
+                    TestsUtil.getAnonymousGlobalAdminRestClient(function(anonymousGlobalClient) {
+                        OrgUnitTestUtil.assertExportOrgUnitFails(anonymousGlobalClient, part.id, null, 401, function() {
+
+                            // Regular users from another application can't export an organisational unit
+                            TestsUtil.generateTestUsers(global.tests.apps.oxford2014, 1, false, function(simon) {
+                                OrgUnitTestUtil.assertExportOrgUnitFails(simon.client, part.id, null, 401, function() {
+
+                                    // Application administrators from another application can't export an organisational unit
+                                    OrgUnitTestUtil.assertExportOrgUnitFails(global.tests.admins.oxford2014.client, part.id, null, 401, function() {
+
+                                        // Regular users from the same application can export an organisational unit
```

Comment:
Documentation error. Given that regular users can't export, line 87 is probably unneccesary